src/Symfony/Component/Form/Extension/Core/DataTransformer/ValueToDuplicatesTransformer.php

```diff
@@ -62,10 +70,13 @@ public function reverseTransform($array)
 
         $result = current($array);
         $emptyKeys = array();
+        $comparator = is_callable($this->comparator) ? $this->comparator : function ($value1, $value2) {
```

Comment:
Can't it also be in the constructor so you just have to use `$this->comparator` below?